src/acceptance-test/java/org/zalando/nakadi/webservice/hila/SubscriptionCursorsAT.java

```diff
@@ -0,0 +1,12 @@
+package org.zalando.nakadi.webservice.hila;
+
+import org.junit.Test;
+import org.zalando.nakadi.webservice.BaseAT;
+
+public class SubscriptionCursorsAT extends BaseAT {
```

Comment:
@lmontrieux Wow, missed it. 
Probably the only reason for this file is to increase "lines added" counter for this pr. Actually I thought to migrate unit test to acceptance test, but then figured out that it won't be simpler. Unfortunately forgot to delete this class